modules/ROOT/pages/procedures.adoc

```diff
@@ -1155,7 +1155,6 @@ For more information, see:
 * link:{neo4j-docs-base-uri}/cypher-manual/current/functions/genai-functions/[Cypher Manual -> GenAI functions]
 * link:{neo4j-docs-base-uri}/genai/tutorials/embeddings-vector-indexes/[GenAI documentation -> Embeddings & Vector Indexes Tutorial]
 
-[role=label--beta]
```

Comment:
They became GA since 5.13. I'm not sure how to reflect this information: should we use `role` attribute or `label:GA[GA from 5.13]`?

Reply:
We don't need this on the `main` branch, on 5.x, we could add a `role` attribute.